tests/seechange_config_test.yaml

```diff
@@ -3,3 +3,6 @@ preloads:
 
 db:
   host: seechange_postgres
+
+subtraction:
+  method: testing_testing
```

Comment:
Wouldn't this be problematic when we actually run tests with subtractions? Or should we just explicitly test each method by setting this parameter when we setup the pipeline for each test?

Reply:
We'll have to do something to test each pipline; it may be a different config, or just updating the config at runtime.